libs/preallocate/src/main/java/org/elasticsearch/preallocate/LinuxPreallocator.java

```diff
@@ -13,11 +13,15 @@
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 
-final class LinuxPreallocator implements Preallocator {
+final class LinuxPreallocator extends AbstractPosixPreallocator {
+
+    LinuxPreallocator() {
+        super(144, 48, 64);
```

Comment:
++
Maybe introduce some named constants?

Reply:
I didn't want to create constants since they would live forever on the class, and *also* as instance members on the superclass. For now I'll add a comment explaining where they come from. In a followup I'll make it more clear in NativeAccess (eg a PosixConstants class).